app/src/main/java/com/kickstarter/ui/viewholders/projectcampaign/VideoElementViewHolder.kt

```diff
@@ -117,6 +127,25 @@ class VideoElementViewHolder(
         playersMap[bindingAdapterPosition] = player
     }
 
+    private fun getMediaSource(videoUrl: String): MediaSource {
```

Comment:
With the migration to media3 exoplayer this piece will actually disappear, but in order to fix the bug asap adding it for now and not extracted into a reusable piece (exactly the same code lives in `VideoPlayerViewer`) 😿